src/roll_renderer.pyj

```diff
@@ -462,6 +462,16 @@ class Beyond20RollRenderer:
                 elif request.name == "Toll the Dead":
                     ttd_dice = v'await self.queryGeneric(request.name, "Is the target missing any of its hit points?", {"d12": "Yes", "d8": "No"}, "ttd_dice", ["d12", "d8"])'
                     damages[0] = damages[0].replace("d8", ttd_dice)
+                
+                # Ranger Ability Support
+                for dmgIndex, dmgType in enumerate(damage_types):
+                    if damage_types[dmgIndex] in ["Colossus Slayer", "Planar Warrior"]:
+                        dVal = damages[dmgIndex].toString()
+                        if dVal:
+                            dmg_dice = v'await self.queryGeneric(request.name, ("Add %1 damage?").replace(/%1/, dmgType), {"0": "No", [dVal]: "Yes"}, "dmg_dice", ["0", dVal])'

```

Comment:
It makes more sense here to do `"Add " + dmgType + " damage"` than using the `%1` and a `replace`